examples/webpack.config.js

```diff
@@ -0,0 +1,55 @@
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+const babelLoaderOptions = {
+  presets: ['@babel/env', '@babel/react'],
+  plugins: ['@babel/plugin-proposal-class-properties']
+};
+
+module.exports = {
+  devServer: {
+    port: 3000,
+    open: true,
+    inline: true,
+    hot: true,
+    watchOptions: {
+      ignored: /node_modules/
+    },
+    historyApiFallback: true
+  },
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
+        use: {
+          loader: 'babel-loader',
+          options: babelLoaderOptions
+        }
+      },
+      {
+        test: /\.css$/,
```

Comment:
Most of the config sections can be shared between dev/prod. Consider importing common parts to reduce duplication.